### DISCUSSION TITLE
ddot equivalent in typst

### DISCUSSION THREAD

**Discussion by sindzicat** [typst/typst#3766] [Q&A]
Hello all!

In LaTeX I type `\ddot x` to get $\ddot x$ and `\dot x` to get $\dot x$. How to type this in typst? I didn't found anything in typst docs.

**Comment by Doublonmousse** [comment]
https://typst.app/docs/reference/math/accent/

**Reply by Doublonmousse** [reply]
![Screenshot_20240322-184234_ASUS_Launcher (1)](https://github.com/typst/typst/assets/115779707/98bfeaca-35cc-4ac2-a97c-a91c53894760)

Just do `$ dot(x) $` or ` $ dot.double(x) $`

**Reply by sindzicat** [reply]
Sorry, you mean `accent(x, dot)` and `accent(x, dot.double)`? So long text?

**Reply by Doublonmousse** [reply]
works too but just `dot(x)` works too. Same for the double one.

I'm wondering if these shorthands are documented somewhere or not though (the long version with accent is, at the very least)

**Reply by sindzicat** [reply]
Ah, I see now! I looked this page before, but only saw example with `accent` command there. I didn't realize I could just use `dot(x)`. Many thanks!

**Comment by PgBiel** [comment]
See the example below:

```typ
$ dot(x) ; dot.double(y) ; dot.triple(y) $
```
![single dot above x, two dots above y, three dots above y](https://github.com/typst/typst/assets/9021226/97a6b519-1ad1-4b66-85cc-693dfe7bc552)

For more information, please refer to:

1. List of built-in Typst symbols: <https://typst.app/docs/reference/symbols/sym/>
    - Note that you can use (paste or through `\u{code here}`) any Unicode character as a symbol, even if it isn't listed above (but not all Unicode characters have nice built-in names, in fact most don't).
2. Detypify (allows you to draw to find a built-in symbol): https://detypify.quarticcat.com/
3. Accent function: <https://typst.app/docs/reference/math/accent/>
    - `dot.double(x)` is implicitly converted to `accent(x, dot.double)`. This is an exception which allows you to "call" the `dot` and `dot.X` symbols as if they were functions.

**Reply by sitandr** [reply]
I would highly recommend using 

```typ
#let ddot = math.dot.double
```

at start of your document if you are using double dot a lot. Name it as you want, maybe like `dot2` or anything.
